examples/lvgl_for_i8080/main/idf_component.yml

```diff
@@ -0,0 +1,4 @@
+dependencies:
+  atanisoft/esp_lcd_ili9488: "^1.0.8"
```

Comment:
this should be declared as:
```
  esp_lcd_ili9488:
    version: "~1.0.0"
    override_path: "../../.."
```

This will avoid unnecessary download of the library since it is already present.